dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/data/TraceWriter.kt

```diff
@@ -6,28 +6,58 @@
 
 package com.datadog.android.tracing.internal.data
 
+import com.datadog.android.rum.GlobalRum
+import com.datadog.android.rum.RumAttributes
+import com.datadog.android.rum.RumErrorSource
 import datadog.opentracing.DDSpan
+import datadog.trace.api.DDTags
 import datadog.trace.common.writer.Writer
+import java.util.Locale
 
 internal class TraceWriter(
     val writer: com.datadog.android.core.internal.data.Writer<DDSpan>
 ) : Writer {
 
+    // region Writer
     override fun start() {
         // NO - OP
     }
 
     override fun write(trace: MutableList<DDSpan>?) {
         trace?.let {
+            it.filter { it.isError }.forEach { span ->
+                sendRumErrorEvent(span)
+            }
             writer.write(it)
         }
     }
 
+    // endregion
+
     override fun close() {
         // NO - OP
     }
 
     override fun incrementTraceCount() {
         // NO - OP
     }
```

Comment:
Shouldn't those be included in the `Writer` region ?

Reply:
Oups, of course :)